forte/processors/stave/stave_processor.py

```diff
@@ -29,8 +29,18 @@
 import collections
 from typing import Dict, Set, Any
 
-from stave_backend.lib.stave_viewer import StaveViewer
-from stave_backend.lib.stave_project import StaveProjectWriter
+
+try:
+    from stave_backend.lib.stave_viewer import StaveViewer
+    from stave_backend.lib.stave_project import StaveProjectWriter
+except ImportError:
+    print(
+        " `stave` is not installed correctly."
+        " Please refer to [extra requirement for stave_processors.py](stave"
+        " extra requirement hyperlink])"
```

Comment:
so are we going to add a hyperlink here?

Reply:
yes. it will be on the next PR as I reported in the https://github.com/asyml/forte/issues/721